sdk/scheduler/src/main/java/com/mesosphere/sdk/api/OfferOutcomeResource.java

```diff
@@ -0,0 +1,34 @@
+package com.mesosphere.sdk.api;
+
+import com.mesosphere.sdk.offer.history.OfferOutcomeTracker;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/v1/debug/offers")
+public class OfferOutcomeResource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(OfferOutcomeResource.class);
+
+    private final OfferOutcomeTracker offerOutcomeTracker;
+
+    public OfferOutcomeResource(OfferOutcomeTracker offerOutcomeTracker) {
+        this.offerOutcomeTracker = offerOutcomeTracker;
+    }
+
+    /**
+     * Renders the current set of offer outcomes as an HTML <table>.
+     * @return HTML response of the table.
+     */
+    @GET
+    public Response getOfferOutcomes(@QueryParam("json") boolean json) {
```

Comment:
Would it not be better to use the Content-Type header field than a query parameter here? Or is this not in keeping with how our APIs currently work?

Reply:
I wanted a way to get the JSON in a browser for testing. Content-type would be better for actually production. Will make that change.